tests/syntactic/ordered_constraints/ordered_constraints.dl

```diff
@@ -0,0 +1,14 @@
+// Tests that constraints should be totally ordered
+
+.decl A(A:number, B:number)
```

Comment:
Could you change the variable names to be different to the relation names, such as by only using caps for the relation names? It would be easier to read with clearly different names.